blog/settings.py

```diff
@@ -1,10 +1,12 @@
 """Django settings for blog project."""
 
 import os
-import dj_database_url
+import environ
 
+env = environ.Env(DEBUG=(bool, True),)
+root = environ.Path(__file__) - 3
```

Comment:
I would call it `root_dir`/`base_dir`. Also, you should not forget to read `.env` file contents into environment (don't forget to gitignore it). See example at: https://github.com/open-craft-guild/crm-smsfly/blob/master/SMSFlyCRM/settings.py#L21